web/assets/main.scss

```
@@ -58,12 +58,31 @@ $govuk-assets-path: "/lpa/frontend/assets/";
   background-color: $govuk-brand-colour;
   margin: 0;
   border: none;
+  width: auto;
+  box-shadow: none;
 }
 
 .app-\!-moj-search__button:hover {
   background-color: $govuk-link-hover-colour;
 }
 
+.app-moj-search__input {
+  border: 1px solid $govuk-border-colour;
+}
+
+.app-search-below-phase-banner {
+  display: block;
+  margin-top: govuk-spacing(2);
+}
+
+.app-search-inline-phase-banner {
+  display: none;
+}
+
+.app-\!-moj-header__navigation-item {
+  border-bottom: 1px solid govuk-colour("white");
+}
```

Can I suggest making this an underline and thickening it on hover (rather than having both underline and border bottom)? It matches how [GDS links work](https://design-system.service.gov.uk/styles/links/) a bit more closely.
```suggestion
.app-\!-moj-header__navigation-item {
  text-decoration: underline;
}

.app-\!-moj-header__navigation-item:hover {
  text-decoration-thickness: 3px;
  text-decoration-skip-ink: none;
}
```